README.md

```diff
@@ -87,6 +87,18 @@ Other interesting flags that can be passed to CMake:
 * `RUN_SYSTEM_TESTS=ON/OFF` toggles building the system tests (i.e. tests requiring 
    an accessible RabbitMQ server instance on localhost), by default this is OFF
 
+## Building RabbitMQ - Using vcpkg
+
+You can download and install RabbitMQ using the [vcpkg](https://github.com/Microsoft/vcpkg) dependency manager:
```

Comment:
Can you format this so it wraps at 80 columns to match the rest of the file.

Here and below.

Reply:
Done, please review.